helm/values.yaml

```diff
@@ -108,6 +108,11 @@ namespacedValidation:
   enabled: false
   mode: ignore  # 'ignore' or 'validate'
 
+# PodSecurityPolicy for Kubernetes cluster using this feature
+psp:
+  enabled: false
+  name:  # name of the psp 
```

Comment:
Not deep into PSPs, but two caveats:
- PSPs have been deprecated. I think we don't mind making backwards compatible changes, but we need to make sure that they won't break the future, so we'd probably want to add a k8s version check to `{{- if .Values.psp.enabled }}` and leave it out for >=1.25 (and maybe warn for >=1.21 [@phbelitz does allow helm that?])
- Afaik, one can specify multiple PSPs, so the `name` field here should be an array, which needs changes to `- {{ .Values.psp.name }}`.

Also, would we want to check for empty names? With a single empty name, installation breaks if enabled=true, which I'd guess is what we want. But if it becomes an array, do we ignore empty lines (if valid ones exist) or still fail due to invalid k8s manifest? Would guess we choose the latter, but I guess @phbelitz is more into that

Reply:
I think I would disagree on the first point. In case someone sets a PSP in v1.25 and above, Connaisseur should not silently drop it, but actually fail. As for the thoughts on issuing a warning, I would keep that to Kubernetes itself and not inherit Kubernetes deprecation.

On the second point, I agree that it should be a list and handled as such in case a list is possible (which I think is correct). I would think that an empty list needs to be handled such that no policy is written in that case or an error is thrown. In case the list contains empty elements, I would suggest to fail if it fails.